### DISCUSSION TITLE
TX16S...  1 + 2 Layout

### DISCUSSION THREAD

**Discussion by mikeyJRB** [EdgeTX/edgetx#2622] [General]
Loaded 2.8 RC3 this morning onto my RM TX16S.  I noticed in Screen Settings when choosing a layout that the 1 + 2 Layout was really just one row of 3 windows.   Whereas the 2 + 1 Layout is 1 large window on the left of the screen and 2 smaller windows on the right.   Shouldn't the 1 + 2 Layout be a mirror of the 2 + 1 Layout?

**Comment by pfeerick** [comment]
>  I noticed in Screen Settings when choosing a layout that the 1 + 2 Layout was really just one row of 3 windows.

No, because there is no need to mirror layouts - what the 'Mirror' layout option is for. Generally, if there is a `+` symbol between them, the naming follows first digit describes number of rows in first column, second digit describes number of rows in second column. But the seperator is a `x` then it's a single column, with equal spacing, The 1+2` is an odd turkey, since it's one column, and not equally spaced zones - as shown by the previews, and is more suited to the portrait orientation screens. If you have a better idea as to naming for all the zones, suggestions are welcome :)
